woocommerce/woocommerce
# Render site thumbnail in CYS intro screen

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/40902.

This PR adds actual rendering of the site when user has created a site using AI

### How to test the changes in this Pull Request:

1. Using a fresh site
1. Make sure to enable `customize-store` feature flag
1. Go to `WooCommerce -> Home` and click `Customize Store` task
2. Observe that no site preview is displayed
3. Go through the AI steps until it completes creating the site
1. Go back to `WooCommerce -> Home` and click `Customize Store` task
1. Observe that the iframe preview is displayed
1. Observe that the admin bar is not displayed in the iframe preview

<img width="1528" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/8a09cbb9-d7c6-4094-b43f-2f515b2b62cf">



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
